schema/commonmeta/commonmeta.schema.json

```diff
@@ -0,0 +1,541 @@
+                        "enum": [
+                            "Abstract",
+                            "Description",
+                            "Summary"
+                        ]
+                    }
+                },
+                "required": [
+                    "description"
+                ]
+            }
+        },
+        "geo_locations": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "geoLocationPlace": {
+                        "type": "string"
+                    },
+                    "geoLocationPoint": {
+                        "$ref": "#/definitions/geoLocationPoint"
+                    },
+                    "geoLocationBox": {
+                        "type": "object",
+                        "properties": {
+                            "westBoundLongitude": {
+                                "$ref": "#/definitions/longitude"
+                            },
+                            "eastBoundLongitude": {
+                                "$ref": "#/definitions/longitude"
+                            },
+                            "southBoundLatitude": {
+                                "$ref": "#/definitions/latitude"
+                            },
+                            "northBoundLatitude": {
+                                "$ref": "#/definitions/latitude"
+                            }
+                        },
+                        "required": [
+                            "westBoundLongitude",
+                            "eastBoundLongitude",
+                            "southBoundLatitude",
+                            "northBoundLatitude"
+                        ]
+                    },
+                    "geoLocationPolygons": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "polygonPoints": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/definitions/geoLocationPoint"
+                                    },
+                                    "minItems": 4
+                                },
+                                "inPolygonPoint": {
+                                    "$ref": "#/definitions/geoLocationPoint"
+                                }
+                            },
+                            "required": [
+                                "polygonPoints"
+                            ]
+                        },
+                        "uniqueItems": true
+                    }
+                }
+            },
+            "uniqueItems": true
+        },
+        "provider": {
+            "description": "The provider of the resource. This can be a DOI registration agency or a repository.",
```

Comment:
is this field for recording which registration body the DOI was registered by? I've also seen 'provider' used to capture the organisation/repository where a dataset is stored.

Reply:
This property is indeed not clearly defined. Ideally it should be the organization that manages the `id` of the resource, e.g. Crossref or DataCite for DOIs. But there might be many other `id` providers, e.g. for life sciences datasets. Happy to discuss more.